unbxdAutosuggest.js

```diff
@@ -2260,7 +2260,10 @@ var unbxdAutoSuggestFunction = function ($, Handlebars, params) {
 			}
 
 			if (!noResults && mainlen > 0) {
-
+				if (mainlen > 0 && sidelen === 0) {
```

Comment:
this case is already handled for suggestions sorted by length. Please add this logic in the else part of the same. 
ex: 
`if (this.options.sortByLength) {
					if (sidelen === 0) {
						mainHtml = mainHtml + '<ul class="unbxd-as-maincontent">' + self['prepareSortedSuggestionsHTML']();
					} else {
						mainHtml = mainHtml + self['prepareSortedSuggestionsHTML']();
					}
				} else {
					if (sidelen === 0) {
						mainHtml = mainHtml + '<ul class="unbxd-as-maincontent">';
					}
				}`
Also, the second class is not required, can be removed.